docs/report.md

```diff
@@ -40,4 +40,9 @@ Here comes a description of our domain model.
 
 ## License
 
-## LLMs, ChatGPT, CoPilot, and others
\ No newline at end of file
+## LLMs, ChatGPT, CoPilot, and others
+A large level model is used for language understanding and generation. We have used ChatGPT and Co-Pilot in this project. We used these to expedite the coding in certain areas. 
+We tried asking ChatGPT when hitting an error that we could not figure out why we recieved. ChatGPT is often helpful with both finding but also explain what the error is to understand _why_ we get the error. This has been great to get a better understanding of what we need to fix to make good and functional code.
```

Comment:
Måske skriv noget om hvordan chatgpt også er god til at bruge til at pege en i en retning at hvad man skal søge videre på, give nogle keywords og så videre. Bare lige for sådan at vise at man ikke kun afhænger af ChatGpt.